_data/contribute.yml

```diff
@@ -5,7 +5,7 @@
+          answer: "<h3>Venue</h3><p>Contributors are welcome to propose and discuss their ideas to develop Python packages for neuroimaging in the <a href='https://nipy.discourse.group'>Nipy Discourse instance</a></p><h3>Infrastructure and workflow</h3><p>Let's first look at the infrastructure and workflow of the site</p><p><ul class='list-group'><li class='list-group-item'><strong>Github</strong>:The site is a repo under the nipy organization on<a href='https://github.com/nipy/nipy.github.com' target='_blank'> github</a>.This means that contribution to it goes by way of forking the repo and doing a pull request (PR).</li><li class='list-group-item'><strong>Github pages</strong>: We also use hosting on github pages, which only means that the site lives on a branch called gh-pages that gets rendered at nipy.github.io.</li><li class='list-group-item'><strong>CI</strong>: We use continuous integration to preview all PRs by way of viewing <a href='https://circle-artifacts.com/gh/vsoch/nipy-jekyll/63/artifacts/0/home/ubuntu/nipy-jekyll/_site/index.html' target='_blank'>artifacts</a>.</li></ul></p><h3>Your Workflow</h3><p>You should do the following:</p><ul><li>Fork the nipy jekyll repo</li><li>Make your changes</li><li>Submit a PR</li><li>We will review the PR, and push to master and ghpages branches.</li></ul></p><h3>Details</h3><ol><li>Add your package info to the data file, in the same format. This will have it rendered automatically in the navigation, and added to the animated brain on all pages.</li><li>Add a base page for your package in the base directory of the site repo. It should be named [packagetag].html where [packagetag] corresponds with the tag variable in the data file. We have a page template for packages, and the easiest thing to do is copy another package's page to start.</li><li>Static site content can be put in the js,css,img folders in the base directory. Any static content can be linked to like '{{ '/img/myimage.png' | prepend: site.baseurl }}'. Keep it organized.</li><li>To contribute an svg image, put it in the '_includes/svg' folder and include like {% include svg/mysvg.svg %}</li><li>Subpages should be contributed as posts to <a href='http://neuroimaging.tumblr.com'>blog content</a>.</li></ol><p>"
```

Comment:
I would make this an external link.

Reply:
Thanks for the review @vsoch :100: ! I do not get how you would propose to change it, though :thinking: .

Reply:
An external link has an `target="_blank"` attribute.
```suggestion
          answer: "<h3>Venue</h3><p>Contributors are welcome to propose and discuss their ideas to develop Python packages for neuroimaging in the <a target="_blank" href='https://nipy.discourse.group'>Nipy Discourse instance</a></p><h3>Infrastructure and workflow</h3><p>Let's first look at the infrastructure and workflow of the site</p><p><ul class='list-group'><li class='list-group-item'><strong>Github</strong>:The site is a repo under the nipy organization on<a href='https://github.com/nipy/nipy.github.com' target='_blank'> github</a>.This means that contribution to it goes by way of forking the repo and doing a pull request (PR).</li><li class='list-group-item'><strong>Github pages</strong>: We also use hosting on github pages, which only means that the site lives on a branch called gh-pages that gets rendered at nipy.github.io.</li><li class='list-group-item'><strong>CI</strong>: We use continuous integration to preview all PRs by way of viewing <a href='https://circle-artifacts.com/gh/vsoch/nipy-jekyll/63/artifacts/0/home/ubuntu/nipy-jekyll/_site/index.html' target='_blank'>artifacts</a>.</li></ul></p><h3>Your Workflow</h3><p>You should do the following:</p><ul><li>Fork the nipy jekyll repo</li><li>Make your changes</li><li>Submit a PR</li><li>We will review the PR, and push to master and ghpages branches.</li></ul></p><h3>Details</h3><ol><li>Add your package info to the data file, in the same format. This will have it rendered automatically in the navigation, and added to the animated brain on all pages.</li><li>Add a base page for your package in the base directory of the site repo. It should be named [packagetag].html where [packagetag] corresponds with the tag variable in the data file. We have a page template for packages, and the easiest thing to do is copy another package's page to start.</li><li>Static site content can be put in the js,css,img folders in the base directory. Any static content can be linked to like '{{ '/img/myimage.png' | prepend: site.baseurl }}'. Keep it organized.</li><li>To contribute an svg image, put it in the '_includes/svg' folder and include like {% include svg/mysvg.svg %}</li><li>Subpages should be contributed as posts to <a href='http://neuroimaging.tumblr.com'>blog content</a>.</li></ol><p>"
```

Reply:
:+1: Thanks @vsoch. Done in 0421028. Push forced.